chat-widget/src/components/livechatwidget/livechatwidgetstateful/LiveChatWidgetStateful.tsx

```diff
@@ -133,6 +133,8 @@ export const LiveChatWidgetStateful = (props: ILiveChatWidgetProps) => {
     useEffect(() => {
         if (state.appStates.conversationState !== ConversationState.Closed) {
```

Comment:
Replace this if statement with one-liner

Reply:
fixed